CHANGELOG.md

```diff
@@ -87,7 +87,6 @@ You may also find the [Update Guide](UPDATING.md) useful.
 - Use widening multiply method for much faster integer range reduction. (#274)
 - `Uniform` distributions for `bool` uses `Range`. (#274)
 - `Uniform` distributions for `bool` uses sign test. (#274)
-- Add `HighPrecision01` distribution. (#320)
```

Comment:
Can we not assume #320 will land before 0.5?

Reply:
We can, but there is some open discussion around using a wrapper type, and adding range code. I now feel like it is better to land `HighPrecision01` in combination with the range code, as that seems like the more common use. And I don't mind if that takes some longer to get right...

Reply:
Okay. I was only thinking that I don't want to make a release which removes functionality without a viable replacement, but there isn't really much removed.